anda/stardust/black-hole/stardust-black-hole.spec

```
@@ -0,0 +1,39 @@
+%global commit 0b847b6ddc383bfcc1e133a2238a37ce8202fe95
+%global commit_date 20240824
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+# Exclude input files from mangling
+%global __brp_mangle_shebangs_exclude_from ^/usr/src/.*$
+
+Name:           stardust-black-hole
+Version:        %commit_date.%shortcommit
+Release:        1%?dist
+Summary:        Spatial storage for stardust xr.
+URL:            https://github.com/StardustXR/black-hole
+Source0:        https://github.com/StardustXR/black-hole/archive/%commit/black-hole-%commit.tar.gz
```

```suggestion
Source0:        %url/archive/%commit/black-hole-%commit.tar.gz
```
don't repeat yourself